src/java.base/unix/native/libnio/MappedMemoryUtils.c

```diff
@@ -43,6 +43,9 @@ typedef char mincore_vec_t;
 jboolean JNICALL MappedMemoryUtils_isLoaded0(JNIEnv *env, jobject obj, jlong address,
                                              jlong len, jlong numPages)
 {
+#ifdef __OpenBSD__
+    return JNI_FALSE;
```

Comment:
Is this like a FIXME, or is this just not supported by OpenBSD? I am not well acquainted with MappedMemoryUtils, but it seem to me that if this is not suppored on OpenBSD, there would be more checks that disable the functionality?

Reply:
Again, I don't really know. It's the port as I've received it, so I'm just posting it for the overview. It's good feedback though. I will have to look into more details when I get to that part, or even drop the specifics for Open/NetBSD for now.